CHANGELOG.md

```diff
@@ -4,9 +4,9 @@
 
 ### Grafana Mimir
 
-* [CHANGE] Store-gateway: change expanded postings index cache key format. #4770
+* [CHANGE] Store-gateway: change expanded postings and postings index cache key format. #4770
```

Comment:
```suggestion
* [CHANGE] Store-gateway: change expanded postings and postings index cache key format. These caches will be invalidated when rolling out the new Mimir version. #4770
```

Reply:
sorry, i didn't realize that's what you meant